src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public int getRandomBall() {
+        ColorSupplier color = new ColorSupplier();
+        System.out.println(color.getRandomColor());
```

Comment:
The `System.out.println` call should not be here. The method is responsible for getting a random Ball, not for printing the color. Printing, if necessary, should be done outside this method where the `getRandomBall` method is called.